.circleci/config.yml

```diff
@@ -43,7 +44,9 @@ jobs:
             pip install --upgrade pip
             pip install Cython
             pip install -e .
-            python .circleci/test_script.py
+            TESTFILES=$(circleci tests glob examples/*.py | circleci tests split --split-by=timings)
+            echo ${TESTFILES}
+            python3 -m unittest ${TESTFILES}
```

Comment:
Why unittest?

Reply:
Again, no particular reason. I just wanted a way to run all the python scripts at once..

Reply:
ok sure